frontend/app-preview/src/components/AppBarConfig/AppPreviewBarConfig.tsx

```diff
@@ -107,7 +107,7 @@ export const appPreviewButtonActions = (
 
   const action: AltinnButtonActionItem[] = [
     {
-      title: 'top_menu.preview_back_to_editing',
+      title: 'top_menu.preview-back-to-editing',
```

Comment:
Is it correct to change this to `-`? 🤔 
We use `_` other places I believe? 😄

Reply:
I agree, but used dash here since it was already used for process-editor 🤔 But I guess all can be underscore!

Reply:
I think we should have all languages written the same way yes 🙌 😄